releases/v21.1.0-alpha.2.md

```diff
@@ -0,0 +1,544 @@
+- Fixed [`cockroach demo --global`](../v21.1/cockroach-demo.html) from crashing with `didn't get expected magic bytes header`. [#58466][#58466] {% comment %}doc{% endcomment %}
+- Previously, for certain log files CockroachDB would both flush individual writes (i.e., propagate them from within the `cockroach` process to the OS) and also synchronize writes (i.e., ask the OS to confirm the log data was written to disk). The per-write synchronization part was unnecessary and, in fact, found to be possibly detrimental to performance and operating cost, so it was removed. Meanwhile, the log data continues to be flushed as previously, and CockroachDB also periodically (every 30s) request synchronization, also as previously. [#58995][#58995] {% comment %}doc{% endcomment %}
+- The parameter `sync-writes` for file sink configurations has been removed. (This is not a backward-incompatible change because the configuration feature is new in v21.1.) [#58995][#58995] {% comment %}doc{% endcomment %}
+- The parameter `buffered-writes` for file sink configurations has been added. It is set to `true` (writes are buffered) by default; and set to `false` (i.e., avoid buffering and flush every log entry) when the `auditable` flag is requested. [#58995][#58995] {% comment %}doc{% endcomment %}
+- The default output format for `file-group` and `stderr` sinks has been changed to `crdb-v2`. This new format is non-ambiguous and makes it possible to reliably parse log files. Refer to the format's [documentation](https://github.com/cockroachdb/cockroach/blob/master/docs/generated/logformats.md) for details. Additionally, it prevents single log lines from exceeding a large size; this problem is inherent to the `crdb-v1` format and can prevent [`cockroach debug zip`](../v21.1/cockroach-debug-zip.html) from retrieving `v1` log files. The new format has also been designed so that existing log file analyzers for the `crdb-v1` format can read entries written the new format. However, this conversion may be imperfect. Refer to the new format's [documentation](https://github.com/cockroachdb/cockroach/blob/master/docs/generated/logformats.md) for details. In case of incompatibility, users can force the previous format by using `format: crdb-v1` in their logging configuration. [#59087][#59087] {% comment %}doc{% endcomment %}
+
+### API endpoint changes
+
+- The notable event `create_statistics` is only reported when the [cluster setting](../v21.1/cluster-settings.html) `sql.stats.post_events.enabled` is enabled. This fact is now also reported in the [event log reference documentation](https://github.com/cockroachdb/cockroach/blob/master/docs/generated/eventlog.md). [#57877][#57877] {% comment %}doc{% endcomment %}
+- The `Timestamp` field of structured notable events is now numeric, and encodes a number of nanoseconds since the unix epoch. [#58070][#58070] {% comment %}doc{% endcomment %}
```

Comment:
```suggestion
- The `Timestamp` field of structured notable events is now numeric, and encodes a number of nanoseconds since the Unix epoch. [#58070][#58070] {% comment %}doc{% endcomment %}
```